k8s/examples/sfc/sfc-vpp.yaml

```diff
@@ -9,7 +9,7 @@ spec:
       type: Pod
       podSelector:
         cnf: vpp-cnf1
-      interface: memif1
+      outputInterface: memif1
```

Comment:
Please do not commit changes in this file.'

Using just `interface` is simpler in case that the pod has only one custom interface.

Reply:
Yah Ok sorry, ignore the commit on this file please.

Reply:
Sorry, I cannot just ignore it, you have to revert the changes in this file, otherwise I cannot merge this PR.

Reply:
Done, confirme me please if it OK?

Reply:
ok ;)